webapp/lib/model.g.dart

```diff
@@ -417,12 +417,43 @@ void listenForUpdates<T>(
   });
 }
 
+/// A snapshot of a document's id and data at a particular moment in time.
 class DocSnapshot {
   final String id;
   final Map<String, dynamic> data;
 
   DocSnapshot(this.id, this.data);
 }
 
+/// A batch update, used to perform multiple writes as a single atomic unit.
+/// None of the writes are committed (or visible locally) until
+/// [DocUpdate.commit()] is called.
+abstract class DocUpdate {
```

Comment:
I wonder if this should be called `DocUpdateBatch`? It wasn't obvious to me from the name that it was really doing batching

Reply:
Good point. `DocUpdateBatch` is more indicative of its function. Renamed.